docs/modules/ow.md

```diff
@@ -145,48 +145,42 @@ Issues a 1-Wire rom select command. Make sure you do the `ow.reset(pin)` first.
 #### Example
 ```lua
 -- 18b20 Example
-pin = 9
+-- 18b20 Example
+pin = 3
 ow.setup(pin)
-count = 0
-repeat
-  count = count + 1
-  addr = ow.reset_search(pin)
-  addr = ow.search(pin)
-  tmr.wdclr()
-until (addr ~= nil) or (count > 100)
```

Comment:
No point of cycling through all sensors in this example. Only first found sensor is read.